protected/humhub/modules/content/models/Content.php

```diff
@@ -780,6 +782,16 @@ public function getContentContainer()
         return $this->hasOne(ContentContainer::class, ['id' => 'contentcontainer_id']);
     }
 
+    public function getPost()
```

Comment:
@gevorgmansuryan I don't think it's a good idea to add relations like `getPost()` or `getActivity()`.

- There are also many other content types like `WikiPage` or `Polls` which cannot be covered by this.
- The relation matches also Activites/Wiki with the Post ID 
- E.g. a Acitivty only query, would also try to load Posts e.g. - and become more complex.

I know such heavy polymorphic relation usage is really hard to map and not very elegant.

As described in the previous PR we want to improve the base of this relations. Maybe we wait for this before adding such relations?